roles/commcare_sync/templates/django/celery_default_supervisor_config.j2

```diff
@@ -3,3 +3,7 @@ command = {{ virtualenv_dir }}/bin/celery_default_start
 user = {{ app_user }}
 stdout_logfile = {{ log_dir }}/celery.log
 stderr_logfile = {{ log_dir }}/celery.log
+stdout_logfile_maxbytes=0
```

Comment:
can you add a note explaining that we are using logrotate and linking to these docs to show why this is done? http://supervisord.org/logging.html#activity-log-rotation

Reply:
sure, will add Cory.

Reply:
Done Cory. Please check.

Reply:
```
Is submodules considered the "right" way to import a third party role? Like there's not something simpler we can do to just reference it?
```
yes, we can do that. shall we do it in that way? As it is simple code which does have much changes I have done in this way.

Reply:
Can we do something like this? https://github.com/dimagi/commcare-cloud/commit/6e71572b15bde810eb8c18153970e492f4ce79dd

Submodules just seem awkward if we can avoid them.

Reply:
sure, I will do it. Thanks Cory.

Reply:
> Can we do something like this? [dimagi/commcare-cloud@6e71572](https://github.com/dimagi/commcare-cloud/commit/6e71572b15bde810eb8c18153970e492f4ce79dd)
> 
> Submodules just seem awkward if we can avoid them.

Updated the playbooks as per your suggestion Cory. Please review. Thanks